src/others/observer/lv_observer.c

```diff
@@ -452,6 +452,8 @@ lv_observer_t * lv_obj_bind_checked(lv_obj_t * obj, lv_subject_t * subject)
 #if LV_USE_LABEL
 lv_observer_t * lv_label_bind_text(lv_obj_t * obj, lv_subject_t * subject, const char * fmt)
 {
+    LV_ASSERT_NULL(obj);
```

Comment:
I suggest
```suggestion
    LV_ASSERT_NULL(obj);
    LV_ASSERT_NULL(subject);
```
here and for the others too.